validation/report.md

```diff
@@ -34,23 +46,28 @@ To run the process and main analysis for the project, the research team extracte
 
 ## Methods
 
-For Phase II validation official datasets compiled by local partners, or official datasets, were analysed for the following cities: Belfast, Hong Kong, Olomouc, and Sao Paulo. The data was then divided into edge data and destination data. Official edge datasets include the urban street network and were compared to the OSM derived street network dataset. Official destination datasets include supermarkets and similar establishments and were compared to the OSM derived food destination dataset. 
+For Phase II validation official datasets compiled by local partners, or official datasets, were analysed for the following cities: Belfast, Hong Kong, Olomouc, and Sao Paulo. The data was then divided into edge data and destination data. Official edge datasets include the urban street network and were compared with the OSM derived street network dataset. Official destination datasets include supermarkets and similar establishments and were compared to the OSM derived food destination dataset. 
+
+Official edge data were collected from Belfast, Hong Kong, and Olomouc. Generally, the official edge data did not include pedestrian paths. Additionally, in Hong Kong’s case, due to out-dated expression of its coordinate values, the edge data x and y coordinates were switched and needed to be corrected for analysis. In Olomouc’s case, the projection metadata in the edge shapefile was inputted incorrectly as EPSG 5513 instead of EPSG 5514, leading the edge data’s geographies to be found in Russia instead of the Czech Republic. These errors in projection support the use of open source data where coordinate reference systems are well-established and well-documented for reusability and replication like OSM. OSM’s data has a format and expectations when inputting metadata that is standard globally. OSM makes analysis of street networks more feasible because it allows for researchers to compare ‘apples with apples’. 
+
+Official destination data were collected from Belfast, Olomouc, and Sao Paulo. The official data for Olomouc and Sao Paulo was in point form. Each point is tied to the centroid of the parcel to which the destination corresponds. The official data for Belfast is expressed in polygon form. The polygon data from Belfast is parcel data to which different relevant destinations are tied.  Because the Belfast dataset is polygon data, and the destination data collected from OSM is point data, the Belfast data were not usable for accessibility analysis. Therefore, the Phase II validation for this project was only conducted using Olomouc and Sao Paulo’s official destination data. Additionally, the Sao Paulo data were edited to exclude farmer’s markets so that the comparison between the Official data and the OSM data was more consistent. 
```

Comment:
Make a note here that the problem with the Belfast data was less that it was parcels (which, if small, could be converted to reasonably accurate centroids), but rather that the parcels were enormous, often the size of entire parks or small neighborhoods. That made it unusable (if I'm remembering correctly).

The word "data" is plural, best to change "was" -> "were" and "is" -> "are" here.